CHANGELOG.md

```diff
@@ -33,8 +37,9 @@
 
 - Initial Release
 
-[Unreleased]: https://github.com/swup/forms-plugin/compare/0.3.1...HEAD
+[Unreleased]: https://github.com/swup/forms-plugin/compare/0.3.2...HEAD
```

Comment:
We can/should remove this one here as well as it's no longer in use.

Reply:
Ah! Nice